core/src/main/java/greencity/constant/SwaggerExampleModel.java

```diff
@@ -14,6 +14,13 @@ public final class SwaggerExampleModel {
         + "\t</ul>\n"
         + "\t<pre>\n";
 
+    private static final String EXAMPLE_FOR_ADD_ECO_NEWS =
+            "  \"title\": \"string\",\n"
+            + "  \"text\": \"string\",\n"
+            + "  \"tags\": [\"valid tag\"],\n"
+            + "  \"source\": \"https://example.org\",\n"
+            + "  \"shortInfo\": \"string\"\n";
+
```

Comment:
this is only used in one place currently (the "add econews" example), so maybe just include it as-is? no need for a separate constant.

also, the `EXAMPLE` variable is now only used in one place as well (the "update econews") example, so would be good to either keep both like this or remove both.